### ISSUE TITLE
Segmentation settings

### ISSUE THREAD

**Issue by huangzizheng01** [wangf3014/Mamba-Reg#3]
Hello! Thank you for your excellent work. I'd like to learn about the settings of the segmentation section, like hyper-parameters and  whether multi-scale test is used. Thanks!

**Comment by JiahaoPlus** [comment]
Thanks for your interest. We use the AdamW optimizer with a weight decay set at 0.01. The models were optimized using a total batch size of $16$. The initial learning rate is $6\times10^{−5}$. Our training schedule includes a linear warmup for the first $1,500$ iterations followed by a linear learning rate decay. The total training duration is set for $160K$ iterations. Data augmentation is applied following standard practices. For evaluation, we use the validation set as in previous papers. We didn't use the multi-scale test.

**Comment by huangzizheng01** [comment]
> Thanks for your interest. We use the AdamW optimizer with a weight decay set at 0.01. The models were optimized using a total batch size of 16. The initial learning rate is 6×10−5. Our training schedule includes a linear warmup for the first 1,500 iterations followed by a linear learning rate decay. The total training duration is set for 160K iterations. Data augmentation is applied following standard practices. For evaluation, we use the validation set as in previous papers. We didn't use the multi-scale test.

Thanks for your kind and detailed reply. Your models really achieve excited scores even without multi-scale test. Btw, how about the Mamba-Reg-Large settings in IN-1K training in your recipe? Does it need larger drop-path rate or other more stronger augmentations or changes compared to the base one? Thanks!

**Comment by wangf3014** [comment]
Hi, Thank you for reaching out. In the pretraining stage we set a 2048 batch size with 2e-4 base learning rate for MambaR-Large. During finetuning, we use 0.6 drop path rate instead of MambaR-Base's 0.4. All other settings are the same to MambaR-Base.

**Comment by huangzizheng01** [comment]
> Hi, Thank you for reaching out. In the pretraining stage we set a 2048 batch size with 2e-4 base learning rate for MambaR-Large. During finetuning, we use 0.6 drop path rate instead of MambaR-Base's 0.4. All other settings are the same to MambaR-Base.

Thanks for your reply, it's clear for reproducing the excellent work. I close this issues.
